modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -192,6 +205,17 @@ Compatibility Matrix
 | image:ROOT:no.png[]
 | image:ROOT:yes.png[]
 | image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
```

Comment:
There's no difference between 3.1 and 3.2 from a compatibility perspective, so I think that means we should be specifying '3.2.0' and '3.2.0 Using Scopes and Collections' rows that match 3.1.0.

Reply:
Done